src/game/objects/har.c

```diff
@@ -1820,8 +1847,7 @@ int har_act(object *obj, int act_type) {
         int opp_id = h->player_id ? 0 : 1;
         object *opp =
             game_state_find_object(obj->gs, game_player_get_har_obj_id(game_state_get_player(obj->gs, opp_id)));
-        if(act_type == ACT_LEFT || act_type == ACT_RIGHT || act_type == (ACT_DOWN | ACT_LEFT) ||
-           act_type == (ACT_DOWN | ACT_RIGHT)) {
+        if(h->inputs[0] == '4' || h->inputs[0] == '6' || h->inputs[0] == '1' || h->inputs[0] == '3') {
```

Comment:
I really like that we're using the HAR's input string for these.

Is there a way to make h->inputs[0] less magical? a well named helper method (getter) might be appropriate.

Reply:
yeah, I could add like `get_last_input(har *har)`

Reply:
Done